regression-test/suites/nereids_p0/delete/delete_mow_partial_update.groovy

```diff
@@ -0,0 +1,59 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite('nereids_delete_mow_partial_update') {
+    sql 'set enable_nereids_planner=true'
+    sql 'set enable_fallback_to_original_planner=false'
+    sql "set experimental_enable_nereids_planner=true;"
+    sql 'set enable_nereids_dml=true'
+
+    def tableName1 = "nereids_delete_mow_partial_update1"
+    sql "DROP TABLE IF EXISTS ${tableName1};"
+
+    sql """ CREATE TABLE IF NOT EXISTS ${tableName1} (
+                `uid` BIGINT NULL,
+                `v1` BIGINT NULL 
```

Comment:
The number of columns is not big enough to trigger partial update? you should create a table with 5 columns, and update 1 value column